src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -60,7 +60,7 @@ class Liberty implements Plugin<Project> {
         }
 
         //Create expected server extension from liberty extension data
-        project.afterEvaluate {
+        project.afterEvaluate { 
```

Comment:
random space but maybe im picky

Reply:
I thought Liberty.groovy was going to be removed from the PR.